internal/logger/message/audit/entry.go

```diff
@@ -29,22 +29,29 @@ import (
 // Version - represents the current version of audit log structure.
 const Version = "1"
 
+// ObjectVersion object version key/versionId
+type ObjectVersion struct {
+	ObjectName string `json:"objectName"`
+	VersionID  string `json:"VersionId,omitempty"`
```

Comment:
s/VersionId/versionId/

ObjectVersion is used in Entry where all fields do not start with a capital letter